lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -333,6 +333,9 @@ def create_changeset(module, stack_params, cfn):
 
             changeset_name = 'Ansible-' + stack_params['StackName'] + '-' + sha1(to_bytes(json_params, errors='surrogate_or_strict')).hexdigest()
             stack_params['ChangeSetName'] = changeset_name
+        else:
+            changeset_name=stack_params['ChangeSetName']
```

Comment:
Needs to be `changeset_name = stack_params['ChangeSetName']` to match PEP8 style of whitespace around operators.